base_classes/NXtransformations.nxdl.xml

```diff
@@ -21,200 +21,343 @@
 #
 # For further information, see http://www.nexusformat.org
 -->
-<definition xmlns="http://definition.nexusformat.org/nxdl/3.1"
-	category="base"
-	name="NXtransformations"
-	type="group"
-	extends="NXobject"
-	ignoreExtraGroups="true"
-	ignoreExtraFields="true"
-	ignoreExtraAttributes="true"
```

Comment:
Something seems to be broken in `nyaml2nxdl` since this is converted to `True` instead of `true` now. @RubelMozumder: any idea?

Reply:
It seems there are also some error in the conversion of the docstring for NXtransformations.

Reply:
Nevermind, this was just because I copied the docstring from nxdl into yaml.